frontend/ui/elements/menu_keyboard_layout.lua

```diff
@@ -90,11 +111,16 @@ local sub_item_table = {
     },
 }
 
-for k, _ in FFIUtil.orderedPairs(VirtualKeyboard.lang_to_keyboard_layout) do
+local layouts_number = 0
+local _keyboard_layouts = G_reader_settings:readSetting("keyboard_layouts") or {}
+for k, __ in FFIUtil.orderedPairs(VirtualKeyboard.lang_to_keyboard_layout) do
+    if _keyboard_layouts[k] == true then
+        layouts_number = layouts_number + 1
```

Comment:
Nitpick probably by my broken english mind, but reading `xyz_number`, I expect it to be some kind of index (i.e. `page_number` = current page number vs. `page_count` = total number of pages).
But here, it feels like it is a count ? So does it mean`selected_layouts_count` ?
(Somehow, I'd had no such remark if it was reading as  `selected_layouts_nb` :) not really sure why I'm fine with _nb as meaning count :/)

Reply:
Done